test/workload_discovery/unix/performance.go

```diff
@@ -0,0 +1,221 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package unix
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test/workload_discovery/unix/util"
+)
+
+const (
+	numJVMs          = 25
+	numRandomProcs   = 75
+	binaryPathLinux  = "/opt/aws/amazon-cloudwatch-agent/bin/workload-discovery"
```

Comment:
nit: This is also declared in the `filepermissions_test.go`. Can we reduce this to a single location?